### DISCUSSION TITLE
Multi-language feat: i want to insert kurdish language to the website

### DISCUSSION THREAD

**Discussion by iamyahia** [photopea/photopea#5880] [Ideas]
Hello everyone, best wishes.
guys I would love it if the photopea website supported other languages since I want to include kurdish.
Please tell us how to manage this situation!

**Comment by camoody1** [comment]
If you would like to help translate Photopea into Kurdish, that would be amazing. Users are able to submit translations of important Photopea phrases through an online tool.

Within Photopea, click More -> Language and then scroll all the way to the bottom of the list and choose Create Translation. On the next window, you will find two options for the Kurdish language that have been started, but are incomplete. Each language requires 793 phrase translations. If you feel comfortable translating for one of the languages, I'm sure the Photopea devs would greatly appreciate your assistance. 🙂

**Reply by iamyahia** [reply]
thank u so much for your information, yes of course inshallah we try to add the Kurdish language to it.

**Comment by photopea** [comment]
Do you mean Sorani or Kurmanci?

You can help us translate Photopea to other languages at https://www.photopea.com/translate/

**Reply by iamyahia** [reply]
**Sorry** I mean: `Kurdish - ( Sorani )`
thank you for your _response_ this link will help me too much, I'll try `as soon as possible` inshallah

**Comment by iamyahia** [comment]
Follow these procedures if you wish to add a new language to the photopea website:

1. open [photopea](photopea.com)
2. in top section click: **More > Language >** _scroll to the bottom of the language and click_: **create translation**.
3. Photopea has already created his translation dashboard, so you don't need to write any PR or clone their project, therefore the processes are really simple; simply choose your language and begin translating '793 phrase'.
